Overview.src.html

```diff
@@ -1022,38 +1023,58 @@ <h3 id="request">Request</h3>
      <li><p>Set <span title=concept-XMLHttpRequest-state>state</span> to
      <i title>headers received</i>.
 
-     <li><p><span data-anolis-spec=dom title=concept-event-fire>Fire an event</span> named <code title=event-xhr-readystatechange>readystatechange</code>.
-    </ol>
+     <li><p><span data-anolis-spec=dom title=concept-event-fire>Fire an event</span> named
+     <code title=event-xhr-readystatechange>readystatechange</code>.
 
-    <p>To <span data-anolis-spec=fetch>process response body</span> for
-    <var title>response</var>, run these subsubsteps:
+     <li><p>If <span title=concept-XMLHttpRequest-state>state</span> is not
+     <i>headers received</i>, return.
```

Comment:
Ah, good catch. You added this because the `readystatechange` event could have changed the state, right?

Small nit: ", then return". I think we're slowly moving to an editorial style where after a conditional we use "then".